app/models/form_profile.rb

```diff
@@ -286,15 +286,15 @@ def vet360_mailing_address
     vet360_contact_info&.mailing_address
   end
 
-  def va_profile_address_hash
-    user.va_profile&.address &&
+  def user_address_hash
```

Comment:
I'm also not sure about calling out that a method is a "hash", should it just be address and return the Hash? A case could probably be made that there should be an Address PRO that takes the hash and returns an object that can also have it's own serializer? I might be taking a purist POV here.

Reply:
Ah yeah, that's a good recommendation. I prob will keep the `hash` designation here for now, but mostly just because that's how a lot of other functions that do similar things in this file are named. This whole file is messy and could use a refactor in general though